src/golang/lib/models/shared/operator/operator.go

```diff
@@ -41,6 +41,7 @@ type ResourceConfig struct {
 	NumCPU          *int    `json:"num_cpus,omitempty"`
 	MemoryMB        *int    `json:"memory_mb,omitempty"`
 	GPUResourceName *string `json:"gpu_resource_name,omitempty"`
+	CudaVersion     *string `json:"cuda_version,omitempty"`
```

Comment:
Any chance we could make this a type called `CudaVersionNumber` or something instead of string - that would serve as builtin documentation for the two cuda versions we support.

Above, it would be:
```
	Cuda11_4_1       CudaVersionNumber  = "11.4.1"
	Cuda11_8_0       CudaVersionNumber  = "11.8.0"
```